dwitter/static/js/scrolling.js

```diff
@@ -80,36 +79,34 @@ window.onload = function() {
 };
 
 function registerWaypoint(iframe) {
-  console.log('Registering ' + iframe.src);
-  var inview = new Waypoint.Inview({
+  // eslint-disable-next-line no-new
+  new Waypoint.Inview({
```

Comment:
Again, this disables the Airbnb ESLint config's rule against calling `new` just for its side effects.